src/components/structures/RightPanel.js

```diff
@@ -3,7 +3,7 @@ Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2017 Vector Creations Ltd
 Copyright 2017, 2018 New Vector Ltd
 Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```

Comment:
While you're here, fancy converting those Vector people's copyrights to the foundation? Michael's would remain, but Vector Creations and New Vector both should have been converted to the Foundation at some point (but we didn't)

Reply:
hmm - that wasn't my understanding of how it worked? I didn't think we ever updated copyrights retrospectively.

Reply:
indeed, but a clarification a while ago lead to (painfully) slow adoption of fixing them: https://github.com/vector-im/element-web/issues/13936

Reply:
huh, TIL